Agent-Install-Examples/eks_lumos_install_python/README.md

```diff
@@ -19,11 +19,14 @@ NOTE: The API `base_url` info can be found when creating the API credentials in
 
 Example of the `config.json` file
 
+NOTE: The`api_base_url` will be provided when creating the API keys.
+
 ```
 {
         "falcon_client_id": "8972331hiusgdaugd987987231213",
         "falcon_client_secret": "cdHJKASDsdad7t8sdsajhkasdasd",
-        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH"
+        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH",
+        "api_base_url": "https://api.us-2.crowdstrike.com"
```

Comment:
Recommend that we keep this consistent and call this `cloud` or `cloud_region`. See https://github.com/CrowdStrike/falcon-integration-gateway/blob/94dba6023807deae786036bed3a025778d4881a0/fig/falcon/api.py#L19

Reply:
how about `base_url` (that was my original name)?.. I really don't like coding the urls into the script as you may have to add/remove/change the urls over time which is why I am using the config.json, which is already known in the falconpy sdk. 

The falconpy sdk uses the term variable `base_url`. Also when creating the keys the UI says `BASE URL` as a copyable item.

What say you @redhatrises?

Reply:
We use base_url in a lot of places, aligns to falconpy as well.